### PR TITLE
Remove guards from modeling commands in the toolbar

### PR DESCRIPTION
**Pull request by franknoirot**
After #4532 they are not working as expected in some cases, and we don't need them anyway, as the command palette selection argument input on each of them will do the job.

## Demo

https://github.com/user-attachments/assets/3a38e9af-d8b1-435c-a2b0-6617333adda3

### CODE REVIEW

File: src/lib/selections.ts [hunk 1]
```diff
@@ -491,6 +491,11 @@ function resetAndSetEngineEntitySelectionCmds(
   ]
 }
 
+/**
+ * @deprecated was made for guards on modeling machine.
+ * Now the selection command bar argument input handles this.
+ * Leaving until we're sure it's not needed elsewhere.
+ */
```
Comment (Irev-Dev) [inline src/lib/selections.ts:498]: These will never get clean up unless you delete them now, we have the PR if we ever need to put them back.

Comment (Irev-Dev) [reply src/lib/selections.ts:498]: The way I check if an exported function is needed is to delete it, then `yarn tsc`, no errors that we're all good.

**Review by Irev-Dev — APPROVED** [review]

File: src/lib/selections.ts (hunk 1 above)
Comment (franknoirot) [reply src/lib/selections.ts:498]: Cool trick I'll use it!

Comment (franknoirot) [reply src/lib/selections.ts]: Good call, I realized that one of them is used within other functions 😅. Leaving that one and removing the other.

File: src/lib/selections.ts [hunk 2]
```diff
@@ -596,6 +611,11 @@ export function canLoftSelection(selection: Selections) {
   )
 }
 
+/**
+ * @deprecated was made for shell guard on modeling machine.
+ * Now the selection command bar argument input handles this.
+ * Leaving until we're sure it's not needed elsewhere.
+ */
```
Comment (pierremtb) [inline src/lib/selections.ts:618]: I added Loft and Shell 2w ago and created these. I say we delete!

Comment (franknoirot) [reply src/lib/selections.ts:618]: Oh of course, I need to delete all of them I searched for the wrong thing

Comment (franknoirot) [reply src/lib/selections.ts:618]: Okay done

**Review by pierremtb — APPROVED** [review]
Tyty this is great

**Comment by pierremtb** [comment]
Triggered CI for you Sir @franknoirot

**Pull request merged**